docs/operator/flux-bootstrap-migration.md

```diff
@@ -126,3 +126,137 @@ spec:
 
 Commit and push the manifest to the Flux repository, and the operator will be automatically upgraded
 when a new Helm chart version is released.
+
+## Migration from Git to OCI artifacts
```

Comment:
I'd add a paragraph somewhere describing the main advantages and motivations of using an OCI artifact over a Git repository.

Reply:
I don’t think this is the right place to explain the reasoning for migrating. This guide is a short how-to doc for migrating from Flux bootstrap to the operator and optionally from Git to OCI. 

An architecture reference (D2) for Flux OCI artifacts would be the doc where we will compare Git to OCI from an UX and security perspective, pro/cons, etc.